openapi/openapi.yaml

```diff
@@ -1206,6 +1213,16 @@ components:
             type: string
       required: [name, views]
 
+    SignificanceTestInfo:
+      type: object
+      properties:
```

Comment:
A few comments:

First, maybe rename `description`->`result_description` and `method`->`method_description`?

Second, this is missing `win_ratio` from the dictionary you define above. But I think this might have been intentional because you wanted this data structure to be more general?

As an alternative to directly adding `win_ratio`, what about adding two fields such as:

```
test_name:
  type: string
  description: the name of the test
test_data:
  type: object
  description: fine-grained data regarding the test
```

`test_name` would always be `pairwise_bootstrap` for now, but if we add other types of tests we could change the name. For now we can leave `test_data` as an unstructured object as it's not used in the interface, but for the paired bootstrap we could include something like:
```
test_data = {
  'system1_name': system1_name,
  'system2_name': system2_name,
  'system1_win_count': wins[0],
  'system2_win_count': wins[1],
  'tie_count': wins[2],
  'n_samples': n_samples,
  'prop_samples': prop_samples,
}
```
This data would be available in a structured format later if it becomes necessary.